### DISCUSSION TITLE
[Bug]: Tab settings not working on 2024.2+

### DISCUSSION THREAD

**Discussion by AyrtonAlbuquerque** [AtomMaterialUI/material-theme-issues#536] [Q&A] [answered]
# Describe the bug

Since the latest version of all IDE's 2024.2+ the Tab Settings stopped working. Tab hightlight has no effect, tab shadow as well and there is this annoying line that separates the tab bar from the editor which was not present using the plugin with the previous versions of the IDE's.

# To Reproduce

Update to new version 2024.2+

# Expected behavior

Tab highlight position working as expected and division line between tab bar and editor to disapear.

# Material Theme Version

Latest

# IDE

All

# System Info

No response

# Logs/Screenshots

In this first screenshot is possible to see that annoying line between the tab bar and the editor. This was not present in the previous versions of all IDE's. My bet is the new ui is messing the tab bar up.
![Tab Bar](https://github.com/user-attachments/assets/12784acd-7057-4262-a8e9-1a4eee3059a3)

Tab highlight position stopped working as well.
![Tab Settings](https://github.com/user-attachments/assets/3e65b5dc-d437-4923-a344-3d0670218a65)

**Comment by mallowigi** [comment, marked as answer]
I know, I already have a fix in hand, but I need to test it a bit more before releasing it.
